CHANGELOG.md

```diff
@@ -3,6 +3,17 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.2.0] - 2019-08-27
```

Comment:
We normally leave the date as "TBD" until you know you're going to do a release (or a separate tiny PR to set the date when you're about to do it)

Reply:
Set to TBD.